frontend/src/components/session-in-progress/question/question.scss

```diff
@@ -28,6 +28,7 @@
     margin-left: 1rem;
     display: flex;
     align-items: center;
+    z-index: 10;
```

Comment:
If this is static `z-index` won't do anything. I don't see anywhere that makes it nonstatic but I could be wrong.